src/cli/index.ts

```diff
@@ -161,6 +161,16 @@ dataCommand
   .argument('<filename...>', 'filename to upload')
   .description('Upload file to project data')
   .action(UploadData);
+dataCommand
+  .command('get')
+  .option('-t, --tag <tag...>')
+  .option('-f, --file <file...>')
+  .option('-b, --before <before>', 'before timestamp')
```

Comment:
Also, what time should the `projects/{projectId}/data/{dataId}` bucket be tagged with... the files uploaded have a `lastModified` added to the metadata, but what about the whole bucket? And should it have maybe a `createdAt` or `lastModified` set, but then date range settable by the user?